### ISSUE TITLE
`-WarningVariable` not working

### ISSUE THREAD

**Issue by Silex** [nightroman/SplitPipeline#29]
Hello,

``` ps1
1..10 | Split-Pipeline -ErrorVariable a -WarningVariable b  -ErrorAction Continue -Script {process{ Write-Warning 'keke'; Write-Error 'kuku' }}
```

`$a` contains the expected errors, but `$b` is empty.

**Comment by nightroman** [comment]
Yes, true... After looking into this, it's the unfortunate effect (+ something else missing) of the change in v1.4.2

> Pipeline runspaces are created with the host used by `Split-Pipeline`. As a
result, some host features can be used by pipeline scripts, e.g. `Write-Host`
and even `Write-Progress`, see `Test-Progress*.ps1` in the project repository.

**Comment by nightroman** [comment]
Now come the questions: how to deals with this, what is more important to keep/lose, what is a possible trade off?

**Comment by nightroman** [comment]
Ah, maybe I have found a reasonable workaround. If `WarningVariable` is specified and there are warnings then the cmdlet calls `WriteWarning` explicitly with each warning. This gets the job done, i.e. the variable is populated **in addition to already written warnings**. This is not perfect but better than before and as good as I can get so far.

**Comment by nightroman** [comment]
Fixed (or rather worked around) in v1.6.2

**Comment by nightroman** [comment]
2.0.1 avoids duplicated warning messages.
